cardano-cli/src/Cardano/CLI/EraBased/Run/Address.hs

```diff
@@ -37,27 +37,19 @@ runAddressKeyGenCmd fmt kt vkf skf = case kt of
   AddressKeyShelleyExtended -> generateAndWriteKeyFiles fmt  AsPaymentExtendedKey  vkf skf
   AddressKeyByron           -> generateAndWriteByronKeyFiles AsByronKey            vkf skf
 
-generateAndWriteByronKeyFiles
-  :: Key keyrole
-#if __GLASGOW_HASKELL__ >= 904
--- GHC 8.10 considers the HasTypeProxy constraint redundant but ghc-9.4 and above complains if its
--- not present.
+generateAndWriteByronKeyFiles :: ()
+  => Key keyrole
   => HasTypeProxy keyrole
-#endif
   => AsType keyrole
   -> VerificationKeyFile Out
   -> SigningKeyFile Out
   -> ExceptT ShelleyAddressCmdError IO ()
 generateAndWriteByronKeyFiles asType vkf skf = do
   uncurry (writeByronPaymentKeyFiles vkf skf) =<< liftIO (generateKeyPair asType)
 
-generateAndWriteKeyFiles
-  :: Key keyrole
-#if __GLASGOW_HASKELL__ >= 904
```

Comment:
I think CPP extension is redundant after that change.

Reply:
Done1